app/experimenter/static/js/tests/AddonForm.test.js

```diff
@@ -358,6 +347,12 @@ describe("The `DesignForm` component for Branched Addons", () => {
       .mockReturnValueOnce(apiResponse)
       .mockRejectedValueOnce(rejectApiResponse);
   };
+  beforeAll(() => {
+    Object.defineProperty(window, "location", {
+      writable: true,
+      value: { replace: jest.fn() },
+    });
+  });
```

Comment:
Is this one not also caught by setup.js?

Reply:
eek i missed one